### PR TITLE
RM-8004 Integrate CoreWeb into Web.Development.WebTesting.TestSite

### PR DESCRIPTION
**Pull request by a-ctor**
https://re-motion.atlassian.net/browse/RM-8004

### CODE REVIEW

File: Remotion/Web/Development.WebTesting.IntegrationTests/App.config [hunk 1]
```diff
@@ -31,7 +31,7 @@
       cleanUpUnmatchedDownloadedFiles="false"
       requestErrorDetectionStrategy="Remotion.Web.Development.WebTesting.IntegrationTests.DiagnosticInformationCollectioningRequestErrorDetectionStrategy, Remotion.Web.Development.WebTesting.IntegrationTests">
     <hosting name="IisExpress" type="IisExpress" port="60401" />
-    <testSiteLayout rootPath=".\..\..\..\..\Development.WebTesting.TestSite">
+    <testSiteLayout rootPath=".\..\..\..\..\Development.WebTesting.TestSite.NetFramework">
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.IntegrationTests/App.config:34]: What is the root path? Shouldn't those be `.Shared`? Except for the root page/global.asax?

File: Remotion/Web/Development.WebTesting.TestSite/Program.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+using System;
+using CoreForms.Web.Infrastructure;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace Remotion.Development.WebTesting.TestSite
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.TestSite/Program.cs:6]:
```suggestion
namespace Remotion.Web.Development.WebTesting.TestSite
```

File: Remotion/Web/Development.WebTesting.TestSite/Web.Development.WebTesting.TestSite.csproj [hunk 3]
```diff
@@ -1,203 +1,27 @@
...
-    <OutputPath>bin\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
-    <NoWarn>1591;0419</NoWarn>
+  <Import Project="..\..\..\Build\Shared.build.props" />
+  <PropertyGroup>
+    <AssemblyName>Remotion.Development.WebTesting.TestSite</AssemblyName>
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.TestSite/Web.Development.WebTesting.TestSite.csproj:8]:
```suggestion
    <AssemblyName>Remotion.Web.Development.WebTesting.TestSite</AssemblyName>
```

Check namespaces in this project.

File: Remotion/Web/Development.WebTesting.TestSite/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "ObjectBinding.Web.Development.WebTesting.TestSite": {
+      "commandName": "Project",
+      "dotnetRunMessages": "true",
+      "launchBrowser": true,
+      "launchUrl": "index.html",
+      "applicationUrl": "https://localhost:50543;http://localhost:50500",
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.TestSite/Properties/launchSettings.json:24]: make sure there's separate ports here from the other test sites?

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/Web/Development.WebTesting.IntegrationTests/App.config (hunk 1 above)
Comment (a-ctor) [reply Remotion/Web/Development.WebTesting.IntegrationTests/App.config:34]: That is the root path for the docker hosting. We did it this way for the object binding projects, but I don't know if it is possible to swap out the folders. Depends on how the build is started

File: Remotion/Web/Development.WebTesting.TestSite/Properties/launchSettings.json (hunk 4 above)
Comment (a-ctor) [reply Remotion/Web/Development.WebTesting.TestSite/Properties/launchSettings.json:24]: I don't think that is necessary. Currently, we don't have a tool to host all of the .NET Core testsite and if we do we should just change the port using environment variables instead

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**